packages/Webkul/Core/src/Helpers/Laravel5Helper.php

```diff
@@ -210,140 +202,74 @@ public function haveProduct(
         return $product;
     }
 
-    /**
-     * Set all session with the given key and value in the array.
-     *
-     * @param array $keyValue
-     */
-    public function setSession(array $keyValue)
-    {
-        session($keyValue);
-    }
-
-    /**
-     * Flush the session data and regenerate the ID
-     * A logged in user will be logged off.
-     *
-     */
-    public function invalidateSession()
-    {
-        session()->invalidate();
-    }
-
-
-    /**
-     * @param array $configs
-     * @param array $productStates
-     *
-     * @return \Webkul\Product\Models\Product
-     */
-    private function haveSimpleProduct(
-        array $configs = [],
-        array $productStates = []
-    ): Product
+    private function haveSimpleProduct(array $configs = [], array $productStates = []): Product
     {
         $I = $this;
-
         if (! in_array('simple', $productStates)) {
             $productStates = array_merge($productStates, ['simple']);
         }
 
         /** @var Product $product */
         $product = $I->createProduct($configs['productAttributes'] ?? [], $productStates);
 
-        $I->createAttributeValues($product->id, $configs['attributeValues'] ?? []);
+        $I->createAttributeValues($product, $configs['attributeValues'] ?? []);
 
-        $I->createInventory($product->id, $configs['productInventory'] ?? []);
+        $I->createInventory($product->id, $configs['productInventory'] ?? ['qty' => 10]);
```

Comment:
As we always fake a valid inventory quantity in the factory (https://github.com/bagisto/bagisto/blob/master/packages/Webkul/Product/src/Database/Factories/ProductInventoryFactory.php), we don't need to overwrite it with a fixed value.

Reply:
i removed createInventory().